src/app/core/components/marios-card/marios-card.component.html

```diff
@@ -0,0 +1,43 @@
+<div
+  class="marios-container"
+  (click)="openDialog()"
+  (keyup)="openDialog()"
+  (mouseenter)="onMouseEnter()"
+  (mouseleave)="onMouseLeave()"
+  tabindex="0" 
+>
+  <div class="col-md-12">
+    <div class="marios-card">
+      <img src="assets/images/person.svg" alt="" />
```

Comment:
Alt text shouldn't be empty, it should be descriptive

Reply:
1c607a567a9fe52411539edf93d0fd314a09410a